webapp/channels/src/components/admin_console/feature_discovery/features/auto_translation.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {defineMessage} from 'react-intl';
+
+import {
+    AdminSection,
+    SectionContent,
+} from 'components/admin_console/system_properties/controls';
+
+import {LicenseSkus} from 'utils/constants';
+
+import './auto_translation.scss';
+import AutoTranslationSVG from './images/auto_translate_svg';
+
+import FeatureDiscovery from '../index';
+
+const AutoTranslationFeatureDiscovery: React.FC = () => {
+    return (
+        <AdminSection>
+            <SectionContent>
+                <div className='AutoTranslationFeatureDiscovery'>
+                    <FeatureDiscovery
+                        featureName='auto-translation'
+                        showSkuTag={true}
+                        minimumSKURequiredForFeature={LicenseSkus.EnterpriseAdvanced}
+                        title={defineMessage({
+                            id: 'admin.auto_translation_feature_discovery.title',
+                            defaultMessage: 'Remove language barriers with Auto-translation',
```

Comment:
```suggestion
                            defaultMessage: 'Remove language barriers with auto-translation',
```

Sorry, my mistake. Shouldn't be capitalized in this sentence